### DISCUSSION TITLE
Notify Birdweather by API

### DISCUSSION THREAD

**Discussion by hyhk** [mmcc-xx/BirdCAGE#70] [Q&A]
Hi all,
Is there a way to notify my Birdweather account by API ?
In fact I don't want to expose by Birdcage to the public internet access so I want to use the Birdweather platform to do that.
I have a Birdweather API ID on my account, but how to update it with Birdcage detections ?

Thanks for your help.
Thanks for your wonderful work on Birdcage.
Regards.

**Comment by matthew73210** [comment]
Need to see if there's a python package that can do that, then get a webpage up to parse the data

**Comment by signal15** [comment]
Anyone figure this out yet?  If I could send a generic webhook, I could just use NodeRed or Zapier to insert it into Birdweather.  Does the Notifications page allow for generic webooks, or only Apprise supported integrations?
